config/config.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+import (
+	gap "github.com/muesli/go-app-paths"
+)
+
+var Scope *gap.Scope = gap.NewScope(gap.User, "trellis")
```

Comment:
I'm going to extract this into its own PR. trellis-cli sort of uses the XDG spec for storing the virtualenv but it's not correct. So now any config/data/cache file usage should go through this package which implements the XDG spec properly.

Reply:
Eh I'm lazy. I'll update the other usage after this